### ISSUE TITLE
Wizard Form Transitions

### ISSUE THREAD

**Issue by va-studio** [ivan-dalmet/formiz#24]
Hi,

Is there a way to create transitions between steps? Currently, I have a custom "submitStep" function to manipulate the visible step. The issue is that it won't work when you trigger the hook "nextStep" with the navigation item, unless with pass some parameters and create some messy js code.

I was wondering if there's an "onChange" hook that is aware of the current and next steps.

Thanks!

**Comment by ivan-dalmet** [comment]
Hi @va-studio,
For now, it did not support animation between steps. But if you have an idea of an API in FormizStep to deal with this, feel free to share it there. So I can update the code :)

**Comment by ivan-dalmet** [comment]
Any chance to do something with the `as` props on the `FormizStep` component? (I'm not used to do animation in React, sorry :( )

**Comment by va-studio** [comment]
Hi @ivan-dalmet I'll have a look at it and let you know. 
As for an idea for the API, I'd be glad to contribute but my knowledge is not enough to do it. 

The workaround I found was to add a class as active to the current step and then fade in the content. Unfortunately is not possible to delay the `display: none` of the previous step, but it looks perfect for its purpose.

I'll let you know if I managed to deal with the `as` props. Meanwhile, feel free to consider this issue closed.

Thanks again for your support! We're about to use Formiz on another project, congrats, you did a really good job here!
